providers/oidc.go

```diff
@@ -126,7 +126,7 @@ func (p *OIDCProvider) RefreshSession(ctx context.Context, s *sessions.SessionSt
 	if s == nil || s.RefreshToken == "" {
 		return false, nil
 	}
-
+	s.Action = "refresh"
```

Comment:
I'm not a fan of setting strings like done here. Either we should change it to a boolean like `s.Refresh = true` or for better extendibility change it to an enum.

Reply:
Furthermore, I would move this logic to the end of the `redeemRefreshToken` method.
Right below:

```golang

	s.AccessToken = newSession.AccessToken
	s.RefreshToken = newSession.RefreshToken
	s.CreatedAt = newSession.CreatedAt
	s.ExpiresOn = newSession.ExpiresOn
	s.SessionType = RefreshedSession

```